src/SignalR/clients/ts/signalr-protocol-msgpack/package.json

```diff
@@ -14,6 +14,7 @@
   "sideEffects": false,
   "scripts": {
     "clean": "rimraf ./dist",
+    "prebuild": "rimraf ./src/pkg-version.ts && node -e \"const fs = require('fs'); const packageJson = require('./package.json'); fs.writeFileSync('./src/pkg-version.ts', 'export const VERSION = \\'' + packageJson.version + '\\';');\"",
```

Comment:
We read the version from the `package.json` and drop it as `export const VERSION = ...` inside the `src` folder before we build. That way the outputs already contain the stamped version